src/ontology/mondo-edit.obo

```diff
@@ -518092,7 +518094,7 @@ xref: UMLS:C4746992 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source=
 is_a: MONDO:0100134 {source="https://github.com/monarch-initiative/mondo/issues/1042", source="https://orcid.org/0000-0001-5208-3432"} ! mitochondrial complex I deficiency, mitochondrial type
 intersection_of: MONDO:0100134 ! mitochondrial complex I deficiency, mitochondrial type
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7458 ! MT-ND3
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7458 {source="OMIM:500014"} ! MT-ND3
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7458 ! MT-ND3
```

Comment:
I do not see a OMIM gene association table for this https://omim.org/entry/500014

Reply:
This information is in the description text. 
This source should be maintained and added to the "protected" file